src/main/java/it/gov/pagopa/pu/debtpositions/controller/InstallmentControllerImpl.java

```diff
@@ -28,4 +31,8 @@ public ResponseEntity<InstallmentDetailDTO> getInstallmentDetail(Long installmen
     return ResponseEntity.ok(installmentService.getInstallmentDetail(installmentId, operatorExternalUserId));
   }
 
+  @Override
+  public ResponseEntity<PagedInstallmentsPaidView> getInstallmentsExport(Long organizationId, String operatorExternalUserId, OffsetDateTime paymentDateFrom, OffsetDateTime paymentDateTo, Long debtPositionTypeOrgId, Pageable pageable) {
```

Comment:
all API related to data export has maximum data export?
if yes, lets:
* change the path in order to have a basePath common to all export API `/export/organization/{organizationId}/installments/paid`
* Let's add as operation description the possibility of an error when the size of the requested export is too big
* rename the operationId into `exportPaidInstallments`